dashboard/app/helpers/levels_helper.rb

```diff
@@ -585,11 +584,6 @@ def blockly_options
     app_options
   end
 
-  private def user_type_from_cookie
-    cookie_key = '_user_type' + (Rails.env.production? ? '' : "_#{Rails.env}")
-    request && request.cookies && request.cookies[cookie_key]
-  end
```

Comment:
https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/layouts/_header.html.haml#L26 This cookie appears to be consumed here as well. Is this still needed?

Reply:
Actually, where does _user_type get set? I can't find it.

Reply:
I think it gets set in https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/config/initializers/devise.rb#L329

Reply:
Ahh yes, thanks.

Reply:
it's at least still used here https://github.com/code-dot-org/code-dot-org/blob/e3b674f877358a8cf30e4772fa8d0df0139c9cfa/lib/cdo/hamburger.rb#L117

Reply:
Yeah, I confused myself into thinking this was the only consumer of `_user_type`. I was wrong. Changes LGTM